src/linkify-matrix.js

```diff
@@ -95,17 +95,26 @@ function matrixLinkify(linkify) {
     S_AT_NAME_COLON_DOMAIN_DOT.on(TT.TLD, S_USERID);
 }
 
+matrixLinkify.onUserClick = function(e, userId) { e.preventDefault(); };
+matrixLinkify.onAliasClick = function(e, roomAlias) { e.preventDefault(); };
+
 matrixLinkify.options = {
-    formatHref: function (href, type) {
+    events: function (href, type) {
```

Comment:
It doesn't "need" it per se. These `options` are just passed to the linkify library. `formatHref` was doing the ID -> path mapping so by not including it you would technically get a path to the ID itself (not a valid path) if you clicked it. This doesn't happen though because we intercept the click (so we can show contextual menus etc). In order to intercept the click you need to specify an `events` option, which is what this is doing.